src/pages/Home/components/ShelterListView/ShelterListView.tsx

```diff
@@ -31,7 +31,9 @@ const ShelterListView = React.forwardRef<HTMLDivElement, IShelterListViewProps>(
       ...rest
     } = props;
 
-    const [searchParams] = useSearchParams();
+    const [filterParamsUrl] = useSearchParams();
+
+    const filterValues = [...filterParamsUrl.values()].filter(Boolean);
```

Comment:
Só para ficar mais semântico 
```suggestion
    const filteredValues = [...filterParamsUrl.values()].filter(Boolean);
```